modules/ckanext-ytp_main/ckanext/ytp/validators.py

```diff
@@ -440,6 +441,18 @@ def validator(key, data, errors, context):
     return validator
 
 
+# FIXME: maybe there is a mistake here? Is the correct order (key, data, errors, context)
```

Comment:
This could be fixed, it only causes confusion. It works correctly because on line https://github.com/vrk-kpa/opendata/blob/b808738a92ab5fa3b879823730fc2c73ea698171/modules/ckanext-ytp_main/ckanext/ytp/validators.py#L439 the order is also wrong.